run_kraken.py

```diff
@@ -22,6 +24,9 @@
 import kraken.pvc.pvc_scenario as pvc_scenario
 import kraken.network_chaos.actions as network_chaos
 import server as server
+from kraken.scenarios.base import Scenario
+from kraken.scenarios.pod import PodScenario
+from kraken.scenarios.runner import ScenarioRunner
 
```

Comment:
Any reason for breaking import style consistency here? previous imports were including the whole module rather than specific classes within the module. Also, this way you're losing namespace for the imported classes.

Reply:
Yes, ALT+Enter, import. :) I use an IDE and don't really look or even see the imports. Is this something we should pay attention to? If yes, is there an autofixer?

Reply:
No hard feeling about this, if the IDE doesn't allow to configure this feature, I won't be the one telling you to not use it :-)

Reply:
![image](https://user-images.githubusercontent.com/86970079/172571913-da0c2a3b-9d09-493b-9662-a3c88171c83d.png)

@sandrobonazzola this is what PyCharm supports. Which of these do you think would be sensible?

Reply:
Above settings looks good to me.
In oVirt we used [isort](https://pycqa.github.io/isort/) to establish common standard for imports.